src/dashboard/cd-dashboard-news.vue

```diff
@@ -164,14 +141,11 @@
       }
 
       &-title {
-        margin: 0;
-
-        &-link {
+        font-weight: bold;
```

Comment:
Wont' that look very... aggressive ? I mean, there is 6 news displayed next to another.
Could we get a screenshot ? :)

Reply:
i'll put one up on the card in a minute